greedy_methods/minimum_waiting_time.py

```diff
@@ -0,0 +1,57 @@
+"""
```

Comment:
Is this a repeat of an existing algorithm in https://github.com/TheAlgorithms/Python/tree/master/scheduling ?

Reply:
Hello @cclauss, I wanted to thank you for your feedback on my previous message. I have incorporated all of the suggested changes, and I want to clarify that my algorithm is not a repeat of any CPU scheduling algorithm. While it can be compared to Shortest Job First, but it is more of a greedy algorithm. Thanks!